### PR TITLE
User counting

### PR DESCRIPTION
**Pull request by hodyroff**
Added user count description according to ownCloud Commercial License.

Backport to 'next' and 10.9

### CODE REVIEW

File: modules/admin_manual/pages/enterprise/installation/install.adoc [hunk 1]
```diff
@@ -55,6 +55,12 @@ upgrade your ownCloud Enterprise to a new version.
 
 TIP: if you want to remove the current license, open ownCloud and navigate to menu:Settings[Admin > General] and click btn:[Remove current license key].
 
+=== User counting
+
+Your ownCloud Enterprise License Key is valid for the purchased amount of users, as per the ownCloud EULA - ownCloud Commercial License.
+When you generate a config.report you will get a counting of the enabled users, the ownCloud guest users (if the guest app is enabled) and the number of user directories created. The later is the user number you will need licenses for. Guest users are currently counted with a fair use policy - which means that you can't have more guest users then you have licensed users. If you exceed the number of licensed users please contact your account manager or ownCloud Partner and true-up as per your contract or EULA with us.
+The user directories are created on first login. When a user is deleted, the user directories are deleted as well. Users which are disblaed count. The last login is irrelavant.
```
Comment (phil-davis) [inline modules/admin_manual/pages/enterprise/installation/install.adoc:62]:
```suggestion
The user directories are created on first login. When a user is deleted, the user directories are deleted as well. Users which are disabled count. The last login is irrelevant.
```

File: modules/admin_manual/pages/enterprise/installation/install.adoc [hunk 2]
```diff
@@ -55,6 +55,12 @@ upgrade your ownCloud Enterprise to a new version.
 
 TIP: if you want to remove the current license, open ownCloud and navigate to menu:Settings[Admin > General] and click btn:[Remove current license key].
 
+=== User counting
+
+Your ownCloud Enterprise License Key is valid for the purchased amount of users, as per the ownCloud EULA - ownCloud Commercial License.
+When you generate a config.report you will get a counting of the enabled users, the ownCloud guest users (if the guest app is enabled) and the number of user directories created. The later is the user number you will need licenses for. Guest users are currently counted with a fair use policy - which means that you can't have more guest users then you have licensed users. If you exceed the number of licensed users please contact your account manager or ownCloud Partner and true-up as per your contract or EULA with us.
```
Comment (phil-davis) [inline modules/admin_manual/pages/enterprise/installation/install.adoc:61]:
```suggestion
When you generate a config.report you will get a counting of the enabled users, the ownCloud guest users (if the guest app is enabled) and the number of user directories created. The latter is the user number you will need licenses for. Guest users are currently counted with a fair use policy - which means that you can't have more guest users than you have licensed users. If you exceed the number of licensed users please contact your account manager or ownCloud Partner and true-up as per your contract or EULA with us.
```

File: modules/admin_manual/pages/enterprise/installation/install.adoc [hunk 3]
```diff
@@ -55,6 +55,12 @@ upgrade your ownCloud Enterprise to a new version.
 
 TIP: if you want to remove the current license, open ownCloud and navigate to menu:Settings[Admin > General] and click btn:[Remove current license key].
 
+=== User counting
+
+Your ownCloud Enterprise License Key is valid for the purchased amount of users, as per the ownCloud EULA - ownCloud Commercial License.
+When you generate a config.report you, will get a counting of the enabled users, the ownCloud guest users -- if the guest app is enabled and the number of user directories created. The latter is the user number you will need licenses for. Guest users are currently counted with a fair use policy -- which means, that you can't have more guest users than you have licensed users. If you exceed the number of licensed users, please contact your account manager or ownCloud Partner and true-up as per your contract or EULA with us.
+The user directories are created on first login. When a user is deleted, the user directories are deleted as well. Users which are disabled count. The last login is not relevant.
```
Comment (mmattel) [inline modules/admin_manual/pages/enterprise/installation/install.adoc:62]:
```suggestion
=== User Counting

Your ownCloud Enterprise License Key is valid for the purchased amount of users, as per the ownCloud EULA - ownCloud Commercial License.
When you generate a config.report, you will get:
* a counting of the enabled users, 
* the ownCloud guest users -- if the guest app is enabled and
* the number of user directories created.

The latter is the user number you will need licenses for. Guest users are currently counted with a fair use policy -- which means, that you can't have more guest users than you have licensed users. If you exceed the number of licensed users, please contact your account manager or ownCloud Partner and true-up as per your contract or EULA with us.
The user directories are created on first login. When a user is deleted, the user directories are deleted as well. Users which are disabled count. The last login is not relevant.
```

**Comment by mmattel** [comment]
@hodyroff is this PR still relevant and should we proceed?

**Comment by hodyroff** [comment]
Yes and yes

File: modules/admin_manual/pages/enterprise/installation/install.adoc [hunk 4]
```diff
@@ -55,6 +55,17 @@ upgrade your ownCloud Enterprise to a new version.
 
 TIP: if you want to remove the current license, open ownCloud and navigate to menu:Settings[Admin > General] and click btn:[Remove current license key].
 
+=== User Counting
+
+Your ownCloud Enterprise License Key is valid for the purchased amount of users, as per the ownCloud EULA - ownCloud Commercial License.
+When you generate a config.report, you will get:
+* a counting of the enabled users,
```
Comment (mmattel) [inline modules/admin_manual/pages/enterprise/installation/install.adoc:62]:
```suggestion
When you generate a config.report, you will get:

* a counting of the enabled users, 
```

**Review by mmattel — APPROVED** [review]

**Pull request merged**